examples/react/basic/src/gen/eliza-ElizaService_connectquery.ts

```diff
@@ -93,20 +96,24 @@ export const ElizaService = {
   },
 } as const;
 
+const queryService = createQueryService({ service: ElizaService });
```

Comment:
I think this will generate uncompilable code with the following service:

```proto
service Locator {
  rpc QueryService(QueryServiceRequest) returns (QueryServiceResponse);
}
```

We have to avoid the collision. I think `$queryService` should be safe.

Reply:
Ooo nice catch. I'll make that modification.